src/public/parts/shells/userSettings/public/index.html

```
@@ -215,7 +215,7 @@ <h2 class="card-title" data-i18n="userSettings.deleteAccount.title"></h2>
 	<!-- 密码确认模态框 -->
 	<dialog id="passwordConfirmationModal" class="modal">
 		<div class="modal-box">
-			<form method="dialog"> <button class="btn btn-sm btn-circle btn-ghost absolute right-2 top-2">✕</button></form>
+			<form method="dialog"> <button class="btn btn-sm btn-square btn-ghost absolute right-2 top-2">✕</button></form>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)